check/controls_test.go

```diff
@@ -1,41 +0,0 @@
-package check
-
-import (
-	"io/ioutil"
-	"testing"
-
-	yaml "gopkg.in/yaml.v2"
-)
-
-const cfgDir = "../cfg/"
-
-// validate that the files we're shipping are valid YAML
-func TestYamlFiles(t *testing.T) {
```

Comment:
Let's not lose this valuable test please (it runs in the automated build and prevents us from shipping a faulty test file)

Reply:
@lizrice the plan is to move all the test to bench-common. But I see your point about testing test files. I'll add the TestYamlFiles back in.

Reply:
Thanks